features/aanschrijfwijze.feature

```diff
@@ -104,11 +105,33 @@ Functionaliteit: Als gemeente wil ik de juiste en consistente aanschrijfwijze va
       | V                      | F. Groenen              |
       | N                      | F. Groenen              |
 
+    Abstract Scenario: Persoon zonder voorvoegsel, zonder adellijke titel/predicaat, zonder actuele en/of ontbonden huwelijk/geregistreerd partnerschap en leeg/onbekend geslachtsnaam
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 999992934 |
+      En de persoon heeft de volgende 'naam' gegevens
+      | naam                           | waarde          |
+      | geslachtsnaam (02.40)          | <geslachtsnaam> |
+      | voornamen (02.10)              | Franklin        |
+      | aanduiding naamgebruik (61.10) | E               |
+      En de persoon heeft nooit een actueel of ontbonden huwelijk of partnerschap gehad
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 999992934                       |
+      | fields              | naam.aanschrijfwijze.naam       |
+      Dan heeft de response 0 personen
```

Comment:
Is het niet vreemd om hier 0 personen terug te krijgen? Zou je niet mogen verwachten hier juist 1 lege personen terug te krijgen:

{
   "personen": [{}]
}

Dan kun je i.i.g. de conclusie trekken dat het genereren van de naam.aanschrijfwijze.naam niet mogelijk is maar dat de betreffende persoon wel bestaat.